e2e/runner/v2_zevm.go

```diff
@@ -103,7 +155,7 @@ func (r *E2ERunner) V2ZEVMToEMVCall(
 	payload []byte,
 	revertOptions gatewayzevm.RevertOptions,
 ) *ethtypes.Transaction {
-	tx, err := r.GatewayZEVM.Call(
+	tx, err := r.GatewayZEVM.Call0(
```

Comment:
Did the function name to use changed randomly after changing the commit used for protocol contracts?

I had this, was a bit annoying, wondering if we could make the overloaded function Go binding more determistic